addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -6,6 +6,7 @@ var Dialog = require('web.Dialog');
 var fonts = require('wysiwyg.fonts');
 var Widget = require('web.Widget');
 var concurrency = require('web.concurrency');
+var session = require('web.session');
```

Comment:
Move this in another PR targeting the first version in which the bug is present.

Reply:
here is the new PR: https://github.com/odoo/odoo/pull/33247  can you assign someone to it ?